$MirOS: ports/devel/pango/patches/patch-pangoxft_pc_in,v 1.1.7.1 2005/03/18 15:45:22 tg Exp $
--- pangoxft.pc.in.orig	Wed Jun 15 00:39:37 2005
+++ pangoxft.pc.in	Thu Sep 22 21:16:51 2005
@@ -7,5 +7,5 @@ Name: Pango Xft
 Description: Xft font support for Pango
 Version: @VERSION@
 Requires: pango pangoft2
-Libs: -L${libdir} -lpangoxft-@PANGO_API_VERSION@ @PKGCONFIG_XFT_LIBS@
+Libs: -L${libdir} -lpangoxft-@PANGO_API_VERSION@ @PKGCONFIG_XFT_LIBS@ -lexpat
 Cflags: -I${includedir}/pango-1.0 @XFT_CFLAGS@
